.github/workflows/gates_report.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       
       - name: Compare gates reports
         id: gates_diff
-        uses: vezenovm/noir-gates-diff@acf12797860f237117e15c0d6e08d64253af52b6
+        uses: noir-lang/noir-gates-diff@1931aaaa848a1a009363d6115293f7b7fc72bb87
```

Comment:
Just updated this to the most recent commit on `noir-lang/noir-gates-diff` main as this still was pointing to my fork.